content/en/software_catalog/use_cases/pipeline_visibility.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Streamline the Development Lifecycle with CI Visibility
+aliases:
+  - /tracing/software_catalog/use_cases/pipeline_visibility
+  - /service_catalog/use_cases/pipeline_visibility
+  - /service_catalog/use_cases/streamlining-development-lifecycle-with-ci-visibility/
+  - /tracing/service_catalog/guides/streamlining-development-lifecycle-with-ci-visibility
+  - /service_catalog/guides/streamlining-development-lifecycle-with-ci-visibility
+  - /tracing/service_catalog/use_cases/pipeline_visibility
+further_reading:
+  - link: "/security/code_security/static_analysis/"
+    tag: "Documentation"
+    text: "Static Analysis"
+  - link: "/dora_metrics/"
+    tag: "Documentation"
+    text: "DORA Metrics"
+---
+
+
+The Software Delivery tab in Software Catalog helps you assess and optimize your service's pre-production status by providing insights into CI pipelines and static analysis violations. You can:
+
+- Monitor the performance of CI pipelines related to your services.
+- Identify security and code quality issues from [Static Analysis][1].
+- Troubleshoot pre-production slowdowns and failures.
+- Track Change Lead Time by integrating with [DORA Metrics][2].
+
+By default, your service is linked to CI pipelines through its repository URL. To modify associated pipelines, update the `ci-pipeline-fingerprints` field in your service metadata.
+
+To add or remove a pipeline associated with your service:
+
+1. Click on your service in [Software Catalog][4] to open the service side panel, and select one of the edit options for Entity Metadata. . 
+
+   **Note**: This is only available for [Software Catalog schema v2.2][3].
+
+   {{< img src="tracing/software_catalog/pipeline-visibility-software-delivery.png" alt="The Delivery tab for monitoring pre-production status in Software Catalog" style="width:100%;" >}}
+
+2. Then, you can either:
+
+   - Find the Software Delivery section, then search for and select the pipelines you want to associate.
```

Comment:
Before looking in the UI, I thought this meant associating pipelines with each other, as opposed to pipelines with the service.
```suggestion
   - Find the Software Delivery section, then search for and select the pipelines you want to associate with the service.
```